WAQS.Server/tools/ttincludes/WAQS.Server.Service.ttinclude

```diff
@@ -1,7 +1,7 @@
 <#@ include file="WAQS.Service.Base.ttinclude"#>
 <#+
 // Copyright (c) Matthieu MEZIL.  All rights reserved.
-    private void WriteServerService(string edmxPath, SpecificationPath[] specificationPathes, SpecificationPath[] dtoPathes, string dtoNamespace, string dtoWCFNamespace, SpecificationPath entitiesPath, string serverFxCommonNamespace, string serverFxEntitiesNamespace, string serverFxDALInterfacesNamespace, string serverFxServiceInterfacesNamespace, string serverFxServiceNamespace, string serverFxSerializableExpressionsNamespace, string serverFxSpecificationsNamespace, string serverEntitiesNamespace, string serverServiceInterfacesNamespace, string serverDALInterfacesNamespace, bool isParallel, string namespaceName, string relativePath)
+    private void WriteServerService(string edmxPath, SpecificationPath[] specificationPathes, SpecificationPath[] dtoPathes, string dtoNamespace, string dtoWCFNamespace, SpecificationPath entitiesPath, string serverFxCommonNamespace, string serverFxEntitiesNamespace, string serverFxDALInterfacesNamespace, string serverFxDALNamespace, string serverFxServiceInterfacesNamespace, string serverFxServiceNamespace, string serverFxSerializableExpressionsNamespace, string serverFxSpecificationsNamespace, string serverEntitiesNamespace, string serverServiceInterfacesNamespace, string serverDALInterfacesNamespace, bool isParallel, string namespaceName, string relativePath)
```

Comment:
Service should not have a dependency on DAL. I guess you do it for RetrySaveChangeException. Please move this class to Fx.DAL.Interfaces